browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -1875,6 +1875,64 @@ index fabd776e617d07a6c30b9f2df30f884491259359..d9c39df7db565545108c765407ce1cda
 -    add_custom_target(Documentation DEPENDS gtkdoc)
 -    WEBKIT_DECLARE_DIST_TARGETS(WPE wpewebkit ${TOOLS_DIR}/wpe/manifest.txt.in)
 -endif ()
+diff --git a/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.inc b/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.inc
+index 1b18e89c26fda2503a8ac617698c324b594b449e..9c77f30898360de0269176f0586eb8ce771b6b97 100644
+--- a/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.inc
++++ b/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.inc
+@@ -14,7 +14,7 @@ static char gDefaultMetallibSrc[] = R"(
+ # 1 "temp_master_source.metal"
+ # 1 "<built-in>" 1
+ # 1 "<built-in>" 3
+-# 415 "<built-in>" 3
++# 382 "<built-in>" 3
+ # 1 "<command line>" 1
+ # 1 "<built-in>" 2
+ # 1 "temp_master_source.metal" 2
+@@ -32,6 +32,7 @@ static char gDefaultMetallibSrc[] = R"(
+ # include <metal_stdlib>
+ 
+ 
++
+ # 1 "./constants.h" 1
+ # 11 "./constants.h"
+ namespace rx
+@@ -1468,6 +1469,7 @@ kernel void generate2DArrayMipmaps(uint lIndex [[thread_index_in_threadgroup]],
+ #include <metal_pack>
+ 
+ 
++
+ # 1 "./format_autogen.h" 1
+ # 11 "./format_autogen.h"
+ namespace rx
+diff --git a/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.metal b/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.metal
+index ae5780d9c77be32891e65a4d6748782de6106c17..cecf1522886b68fe2a996069f9c2c92fcacae058 100644
+--- a/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.metal
++++ b/Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/shaders/mtl_default_shaders_src_autogen.metal
+@@ -11,7 +11,7 @@
+ # 1 "temp_master_source.metal"
+ # 1 "<built-in>" 1
+ # 1 "<built-in>" 3
+-# 415 "<built-in>" 3
++# 382 "<built-in>" 3
```

Comment:
Can we upstream these build fixes?

Reply:
These changes were not actually needed, I commited them by mistake.